README.md

```diff
@@ -33,17 +35,17 @@ a commercial set of Instagram-like filters.
 
 ## How it works
 
-This method is based on
+This method provides an accurate color reconstruction, and is based on
 [three-dimensional lookup tables][wiki-luts] and their two-dimensional
 representation: [hald images][hald-image].
 The core idea is simple: a sample hald image with a uniform color distribution
-is processed using any target software with an unknown color
-transformation algorithm.
+is processed using target color filter with an unknown transformation algorithm.
```

Comment:
is processed using a target color filter with an unknown transformation algorithm